src/release.py

```diff
@@ -61,6 +61,12 @@
 
 LINE = "=" * 80
 
+# Recognize "bug-NNNNNNN", "bug NNNNNNN", and multi-bug variants
+BUG_RE = re.compile(r"\bbug(?:s?:?\s*|-)([\d\s,\+&#and]+)\b", re.IGNORECASE)
```

Comment:
This is loosely based on the equivalent regexp from rob-bugson:

https://github.com/willkg/rob-bugson/blob/cb3493d4a9f1e213807d5b71049258af0fed690c/github-bugzilla-content.js#L11